### PR TITLE
added ukranian lorem ipsum to close issue #9

### PR DESCRIPTION
**Pull request by max-mackie**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h1>Ukrainian Culture</h1>
     </header>
 
     <main class="bg-light">
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nihil sint animi in, sunt assumenda explicabo blanditiis recusandae pariatur autem rerum reiciendis at asperiores doloremque aliquam dolore earum nobis, impedit officiis.</p>
+        <p>Біль сам по собі є великим болем, але головний адипічний Вони ніщо в їхній свідомості, їх слід прийняти, вони пояснюватимуть, лестячи, прийнятих виробників, але відкидаючи обставини, тоді як найжорстокіші та найболючіші з них заважають нам нести будь -які зусилля у виконанні своїх обов’язків. this is Ukranian</p>
```
Comment (fooster82) [inline index.html]: Very nice, just needs a captial letter for 'this' after the Ukrainian text.

**Review by fooster82 — CHANGES_REQUESTED** [review]
Great addition just a minor change.

**Review by cbmcdougall — APPROVED** [review]
Great capitalization! :)

**Review by fooster82 — COMMENTED** [review]
All good

**Pull request merged**